File: jdroid-gradle-commons-plugin/src/main/groovy/com/jdroid/gradle/commons/versioning/IncrementMinorVersionTask.java
package com.jdroid.gradle.commons.versioning;

public class IncrementMinorVersionTask extends AbstractIncrementVersionTask {
	
	public IncrementMinorVersionTask() {
		setDescription("Increments the minor version (X.Y.Z) -> (X.Y+1.0)");
	}

	@Override
	protected void incrementVersion(Version version) {
		version.incrementMinor();
	}

}

File: jdroid-gradle-commons-plugin/src/main/groovy/com/jdroid/gradle/commons/versioning/IncrementMinorVersionTask.kt
package com.jdroid.gradle.commons.versioning

open class IncrementMinorVersionTask : AbstractIncrementVersionTask() {

    init {
        description = "Increments the minor version (X.Y.Z) -> (X.Y+1.0)"
    }

    override fun incrementVersion(version: Version) {
        version.incrementMinor()
    }
}
